src/docs/ftw-styling/how-to-customize-ftw-styles/index.md

```diff
@@ -169,29 +237,27 @@ For example, our default font is defined as:
 }
 ```
 
-And created property set can be used as:
+And created property set were used as:
 
 ```css
 p {
   @apply --marketplaceDefaultFontStyles;
 }
 ```
 
+</extrainfo>
+
 > ⚠️ **Note**: template app is following a pattern where the height of
 > an element should be divisible by **6px** on mobile layout and **8px**
 > on bigger layouts. This affects line-heights of font styles too.
 
-> ⚠️ **Note**: the **@apply** rule and custom property sets most likely
-> won't get any more support from browser vendors as the spec is yet
-> considered deprecated and alternative solutions are being discussed.
-
 ## Fonts
 
-**marketplaceDefaults.css** and **propertySets.css** files are mostly
-responsible of what font styles are used. The font-family itself is
-defined in CSS Property `--fontFamily` and by default, FTW templates use
-Poppins. This is a Google Font, but for performance reasons we have
-served them from Sharetribe's CDN.
+**marketplaceDefaults.css** file is mostly responsible of what font
+styles are used. The font-family itself is defined in CSS Property
```

Comment:
Does CSS Property refer to a variable, CSS Property Sets or properties in CSS (
as in: 
```
#box {
  width: 200px; /* the property is "width" (without the colon) */
}
```
)